styles.css

```diff
@@ -6,6 +6,7 @@
 @import url(card.css);
 @import url(images.css);
 @import url(input.css);
+@import url(Text-Utilies.css);
```

Comment:
follow a consistent naming convention.
`text-utilities.css` would be better here

Reply:
will take care from next time